app/src/main/java/eu/kanade/tachiyomi/util/chapter/ChapterTrackSync.kt

```diff
@@ -40,3 +48,68 @@ fun syncChaptersWithTrackServiceTwoWay(db: DatabaseHelper, chapters: List<Chapte
         }
     }
 }
+
+private var trackingJobs = HashMap<Long, Pair<Job?, Int?>>()
+
+/**
+ * Starts the service that updates the last chapter read in sync services. This operation
+ * will run in a background thread and errors are ignored.
+ */
+fun updateTrackChapterMarkedAsRead(
+    db: DatabaseHelper,
+    preferences: PreferencesHelper,
+    newLastChapter: Chapter?,
+    mangaId: Long?,
+    fetchTracks: (suspend () -> Unit)? = null,
+    delay: Long = 0
```

Comment:
Id switch these around, and make delays default 3000, since only notification is using 0. Flipping them around also allows updateTrackChapterMarkedAsRead(...) { fetchtracks }
```suggestion
    delay: Long = 3000,
    fetchTracks: (suspend () -> Unit)? = null,
```

Reply:
Done